docs/source/release_notes/release-notes.rst

```diff
@@ -49,12 +55,8 @@ develop
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1720>`__)
 
     *    - |fixed|
-         - Fixed an issue where a ``MultipleRunningTimestampServicesError`` would be ignored, resulting in a state where two timestamp services would be able to simultaneously hand out timestamps. Also changed the logic for increasing the timestamp bound when the allocation buffer is exhausted.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1718>`__)
-
-    *    - |fixed|
          - KVS migrations where timestamp data was co-located with AtlasDB data now respect the timestamp service contract.
```

Comment:
The wording here is hard to parse. I would say this like 

"KVS migrations now maintain the guarantee of the timestamp service to hand out monotonically increasing timestamps. Previously, we would reset the timestamp service to 0 after a migration, but now we use the correct logical timestamp.

Note: this does not affect users who are using the Timelock server"